living_documentation_generator/action/action_inputs.py

```diff
@@ -73,3 +74,10 @@ def validate_inputs(self, repositories_json: str) -> None:
         # Validate GitHub token
         if not self.__github_token:
             raise ValueError("GitHub token could not be loaded from the environment")
+
+    def _make_absolute_path(self, path):
```

Comment:
This method should be static / not using self par.

Reply:
Addressed in commit b7557316ddb3b8eb713801b079194d63ba425f14.